about.html

```diff
@@ -0,0 +1,114 @@
+                  </svg>@stella.garon</li>
+            </ul>
+            <div class="row">
+                <div class="col-4">
+                    <img src="/assets/wildflowers.jpg" class="img-fluid">
+                    <p></p>
+                </div>
+                <div class="col-8">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                </div>
+            </div>
+            <h3 class="title text-left">experience</h3>
+            <ul>
+                <li>Lorem ipsum (2020-2023)</li>
+                <li>Lorem ipsum (2017-2019)</li>
+            </ul>
+            <h3 class="title text-left">education</h3>
+            <ul>
+                <li>Lorem ipsum (2015)</li>
+                <li>Lorem ipsum (2019)</li>
+                <li>Lorem ipsum (2023)</li>
+            </ul>
+            <div class="d-grid gap-2">
+                <a class="btn btn-dark" href="contact.html" role="button">contact</a>
+                <p></p>
+            </div>
+        </div>
+    </main>
+        
+    <footer class="container-fluid">
+        <div class="row text-bg-dark" data-bs-theme="dark">
+            <div class="col-6">
+                <p><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-moon-stars-fill" viewBox="0 0 16 16">
+                    <path d="M6 .278a.768.768 0 0 1 .08.858 7.208 7.208 0 0 0-.878 3.46c0 4.021 3.278 7.277 7.318 7.277.527 0 1.04-.055 1.533-.16a.787.787 0 0 1 .81.316.733.733 0 0 1-.031.893A8.349 8.349 0 0 1 8.344 16C3.734 16 0 12.286 0 7.71 0 4.266 2.114 1.312 5.124.06A.752.752 0 0 1 6 .278z"/>
+                    <path d="M10.794 3.148a.217.217 0 0 1 .412 0l.387 1.162c.173.518.579.924 1.097 1.097l1.162.387a.217.217 0 0 1 0 .412l-1.162.387a1.734 1.734 0 0 0-1.097 1.097l-.387 1.162a.217.217 0 0 1-.412 0l-.387-1.162A1.734 1.734 0 0 0 9.31 6.593l-1.162-.387a.217.217 0 0 1 0-.412l1.162-.387a1.734 1.734 0 0 0 1.097-1.097l.387-1.162zM13.863.099a.145.145 0 0 1 .274 0l.258.774c.115.346.386.617.732.732l.774.258a.145.145 0 0 1 0 .274l-.774.258a1.156 1.156 0 0 0-.732.732l-.258.774a.145.145 0 0 1-.274 0l-.258-.774a1.156 1.156 0 0 0-.732-.732l-.774-.258a.145.145 0 0 1 0-.274l.774-.258c.346-.115.617-.386.732-.732L13.863.1z"/>
+                  </svg>© 2023 Company, Inc</p>
+            </div>
+            <div class="col-3">
+                <p></p>
+            </div>
+            <div class="col-3">
+                <a href="twitter.com" class="link-light"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-twitter" viewBox="0 0 16 16">
```

Comment:
For external links, I recommend using the `target='_blank'` property. This will open the page in a new tab instead to replace your current page.